src/Cdn/Cdn.Test/ScenarioTests/Common.ps1

```diff
@@ -118,4 +118,17 @@ function Assert-CompressionTypes($types1, $types2)
             throw "Compression content not equal. " + $value1 + " cannot be found in second array"
         }
     }
+}
+
+<#
+.SYNOPSIS
+Sleep in record mode only
+#>
+function SleepInRecordMode ([int]$SleepIntervalInSec)
+{
+    $mode = $env:AZURE_TEST_MODE
+    if ( $mode -ne $null -and $mode.ToUpperInvariant() -eq "RECORD")
+    {
+        Sleep -Seconds $SleepIntervalInSec 
```

Comment:
Please use Wait-Second instead: https://github.com/Azure/azure-powershell/blob/master/tools/ScenarioTest.ResourceManager/Common.ps1#L307 rather than create your own function.  Fixing this should not require rerecording.

Reply:
Updated